### ISSUE TITLE
How to pass configuration to react-docgen?

### ISSUE THREAD

**Issue by bensampaio** [zeplin/cli-connect-react-plugin#15]
I want to use the `findAllComponentDefinitions` resolver for `react-docgen` but I don't understand where can I configure this. In my components.json file I tried the following:

```json
"plugins": [
    {
      "name": "@zeplin/cli-connect-react-plugin",
      "config": {
        "resolver": "findAllComponentDefinitions"
      }
    }
  ],
```

But this didn't change anything. I still get this error: `Multiple exported component definitions found`.

**Comment by yuqu** [comment]
Hi Bruno, thanks for the feedback. I'll be adding the ability to pass react-docgen configuration parameters from CLI after I make sure that the code generation template could handle multi components at once.

**Comment by yuqu** [comment]
Fixed in v1.1.0
